### DISCUSSION TITLE
유니티 게임씬에서 카메라 회전이 되지 않습니다.

### DISCUSSION THREAD

**Discussion by IndiaInk10** [naverz/zepeto-studio-kor#1120] [5. Q&A]
데스크탑에선 멀쩡한데, 노트북에서 해당 문제가 발생하네요...
UI 영역 확인 및 해상도 설정도 해봤지만, 해당 부분이 문제의 문제가 아니였습니다...
혹시 해결방법 아시는 분은 공유 부탁드립니다.

![image](https://user-images.githubusercontent.com/49700162/182602189-51782326-52e6-4351-b60b-dde850a6a69f.png)

**Comment by chlguswls32** [comment]
IndianInk10님 안녕하세요.
pc에서 되는것이 노트북에서만 안되다니... 왜그럴지궁금하네요. 정확한 해결방법은아니지만

unity debug raycast 쪽기능을 이용하셔서 pc때와 노트북때의 ray가 어떻게 다른지 디버깅 해보시면서 해결방법 찾아보시면\
좋을것같습니다.

참고 유튜브링크 남깁니다.
https://youtu.be/USwrSUpIgLI

**Comment by zlmrzlo** [comment]
저도 이런 문제를 겪은 적이 있어서 댓글 남깁니다.
그전에 확인할 사항이 있는데 혹시 Canvas 중에서 world space 속성으로 만들어 둔 오브젝트가 있는지 궁금합니다.
그것으로 인해서 화면에 클릭이 되던 부분이 클릭이 안되는 경우도 존재하더라구요.
